spec/researches/findKeywordInPageitleSpec.js

```diff
@@ -29,7 +29,7 @@ describe( "Match keywords in string", function() {
 
 		var mockPaper = new Paper( "", { keyword: "focus keyword", title: "focus-keyword" } );
 		result = pageTitleKeyword( mockPaper );
-		expect( result.matches ).toBe( 0 );
+		expect( result.matches ).toBe( 2 );
```

Comment:
This behavior is interesting. First, there's fact that the hyphen is ignored - we should think about whether that is desirable. Second, there now are two matches, I suppose one for "focus" and one for "keyword". I don't think this is particularly problematic for the `pageInTitle` assessment, since here we're not interested in the number of matches. However, this sort of behavior might cause trouble in assessments where the number of matches is important. So I'd start thinking about how this can be avoided already at this step.

Reply:
First part: I just replicated the behavior of Google.
<img width="829" alt="screen shot 2018-06-08 at 11 15 21" src="https://user-images.githubusercontent.com/36954173/41150030-567e5654-6b0d-11e8-91df-77e560f594e7.png">
Second part: The problem is that we really do not know how Google treats multi-word matches. I believe that longer (multi-word) matches get higher weight in Google than shorter (e.g., single-word) matches. But without quotation marks both partial and full matches are definitely considered. Maybe we can rather think about making our assessments somewhat smarter in this respect.